build.gradle

```diff
@@ -53,24 +51,6 @@ jar {
     }
 }
 
-springBoot {
-    buildInfo {
-        properties {
-            artifact = rootProject.name
-            group = 'uk.gov.justice.probation'
-            name = rootProject.name
-            version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
-            time = Instant.now().toString()
-            additional = [
-                    by                   : System.properties['user.name'],
-                    operatingSystem      : "${System.properties['os.name']} (${System.properties['os.version']})",
-                    continuousIntegration: System.getenv('CI') ? true : false,
-                    machine              : InetAddress.localHost.hostName
-            ]
-        }
-    }
-}
```

Comment:
provided by the plugin

Reply:
The DeploymentLogger file uses the name and version, would this break? I'm curious because i've not uses buildInfo properties before

Reply:
https://github.com/ministryofjustice/dps-gradle-spring-boot/blob/acc67135b87cfe0b1d0f1f6e6321862e38339980/src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/pluginmanagers/SpringBootPluginManager.kt#L35

Reply:
Excellent review by the way! 
I think it will be OK, we will find out on startup anyway.

Reply:
`2024-09-26 09:47:33.571  INFO 16194 --- [  restartedMain] u.g.j.p.c.application.DeploymentLogger   : Starting court-case-matcher 2024-09-26 using Java 21.0.1 on MJ004359 |  `
This is the output when running locally, seems fine to me